docs/atlascli/command/atlas-accessLogs.txt

```diff
@@ -12,7 +12,7 @@ atlas accessLogs
    :depth: 1
    :class: singlecol
 
-Manage the access logs of a cluster.
+Retrieve the access logs of a cluster.
```

Comment:
Nit to keep these in line with the language we used while refactoring API docs since we're making changes anyway :)
```suggestion
Return the access logs for a cluster.
```